packages/camera/camera_avfoundation/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 0.9.17+8
```

Comment:
I haven't seen 2 released in a single PR before. This is probably not a standard practice (@stuartmorgan will know)

May worth splitting into 2 PRs, which can make the review easier/faster.

Reply:
CI won't allow this; it can either be two PRs, or it can be one PR with a single version bump.

Reply:
So this PR was meant to be merged after https://github.com/flutter/packages/pull/8351 (it includes changes from that PR), that's why there are two new versions. So either we merge them one after the other or we close #8351 and only merge this one. Sorry for making it confusing.

Reply:
There's a trick that you can set base branch your previous branch first, and after that gets landed, change the base branch back to main.

Reply:
Good idea, but I believe that I can only set the base branch as a branch that exists in the main repo, it doesn't list branches from my fork

Reply:
oh i will review after the previous PR is in then. otherwise it's hard to tell which part is new change.